internal/proto/milvus.proto

```diff
@@ -176,6 +176,8 @@ message DescribeCollectionResponse {
   repeated common.KeyDataPair start_positions = 10;
   // The consistency level that the collection used, modification is not supported now.
   common.ConsistencyLevel consistency_level = 11;
+  // The collection name
+  string collection_name = 12;
```

Comment:
@xiaofan-luan you mentioned this in the issue but after some look at the protos I realized the `CreateCollectionRequest.schema` is the same as `DescribeCollectionResponse.schema`
And the schema contains the original name. If we want to be very clear we could add this field but it would be redundant

`DescribeCollectionResponse.schema.name == DescribeCollectionResponse.collection_name`